tests/authentication/04-check-keys-remote.t

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#-------------------------------------------------------------------------------
+# Checks remote ZMQ keys are created and deleted on shutdown.
+. "$(dirname "$0")/test_header"
+set_test_remote_host
+set_test_number 4
+init_suite "${TEST_NAME_BASE}" <<'__SUITE_RC__'
+[cylc]
+    cycle point format = %Y
+[scheduling]
+    initial cycle point = 2020
+    [[graph]]
+        R1 = t1
+[runtime]
+    [[t1]]            
+        script = sleep 60
+        [[[remote]]]
+            host = $CYLC_TEST_HOST
+__SUITE_RC__
+
+run_ok "${TEST_NAME_BASE}-validate" \
+    cylc validate "${SUITE_NAME}" -s "CYLC_TEST_HOST=${CYLC_TEST_HOST}"
+suite_run_ok "${TEST_NAME_BASE}-run" cylc run "${SUITE_NAME}" \
+    -s "CYLC_TEST_HOST=${CYLC_TEST_HOST}"
```

Comment:
`-s blah` will have no effect here as it sets Jinja2 variables, not shell variables, and the suite doesn't use Jinja2.  `CYLC_TEST_HOST` is set by `test_header` from global config.

Reply:
Ah great, thanks. Think I need to spend a bit more time writing suites!